### ISSUE TITLE
JideTech ONVIF Camera - 

### ISSUE THREAD

**Issue by jdesai61** [dmitrif/onvif2mqtt#24]
I was playing around with my JideTech cameras and notices that the "simpleItem" they send in motion event is an array (see debug below)
```
[1609711795443] INFO  (ONVIF): ONVIF  onSubscriberEvent {"topic":{"_":"tns1:RuleEngine/CellMotionDetector/Motion","$":{"Dialect":"http://www.onvif.org/ver10/tev/topicExpression/ConcreteSet"}},"message":{"message":{"$":{"UtcTime":"2021-01-03T22:09:55.676"},"source":{"simpleItem":[{"$":{"Name":"Window","Value":0}},{"$":{"Name":"Source","Value":"000"}},{"$":{"Name":"VideoSourceConfigurationToken","Value":"VideoSourceToken"}},{"$":{"Name":"VideoAnalyticsConfigurationToken","Value":"VideoAnalyticsToken"}},{"$":{"Name":"Rule","Value":"MyMotionDetectorRule"}}]},"data":{"simpleItem":[{"$":{"Name":"Motion","Value":1}},{"$":{"Name":"State","Value":true}},{"$":{"Name":"IsMotion","Value":true}}]}}}}
```
Is this allowed in ONVIF standard? If so, it will be nice to have a fix.  For now, I hacked SubscriberGroup.js line from
```
    const eventValue = event.message.message.data.simpleItem.$.Value;
```
to
```
   const eventValue = event.message.message.data.simpleItem[0].$.Value;
```
And it worked for my camera - obviously this breaks other cameras that don't send an array. Perhaps a check on testing for array could be added as a fallback? Thanks

**Comment by ksupipr** [comment]
Hi, specification. message.data may be array
https://www.onvif.org/specs/core/ONVIF-Core-Specification.pdf

I fix it make from simpleItem array to object like {$.Name, $.Value}
Check for old cameras and create pull request

**Comment by jdesai61** [comment]
@ksupipr I looked at your branch and didn't see any changes or pull requests. Am I missing something? Thanks

**Comment by ksupipr** [comment]
> @ksupipr I looked at your branch and didn't see any changes or pull requests. Am I missing something? Thanks

Holidays ) 
Create branch tomorrow, write comment with link here

**Comment by ksupipr** [comment]
@jdesai61  https://github.com/ksupipr/onvif2mqtt/tree/bugfix/issue24

I think what value send to the event, for compatibility with old projects.
Now send full object (key: value)

**Comment by dmitrif** [comment]
Resolved with #34 . Reopen if not, thanks!
